### PR TITLE
Release/1.1.8

### PR DESCRIPTION
**Pull request by robincornett**
Milestone: https://github.com/awesomemotive/edd-conditional-success-redirects/milestone/5?closed=1

| Issue # | Description |
|---------|-------------|
| #27 | Use ExtensionLoader to boot plugin |
| #25 | Set PHP minimum to 5.4 |
| #24 | Update EDD_License instantiation to use ExtensionRegistry if available |
| #22 | EDD_Conditional_Success_Redirects class is dynamically declaring properties |
| #20 | Warning: count(): Parameter must be an array or an object that implements Countable |

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -4,6 +4,13 @@ Allows per-product confirmation pages on successful purchases.
 
 == Changelog ==
 
+= 1.1.8 - December 9, 2021 =
```
Comment (robincornett) [inline readme.txt:7]: I went ahead and put Thursday's date in since that's what it's currently scheduled for.

Comment (ashleyfae) [reply readme.txt:7]: I'm out on Thursday, but you're welcome to release it then.

**Review by ashleyfae — APPROVED** [review]

File: readme.txt (hunk 1 above)
Comment (robincornett) [reply readme.txt]: Changing to today then since it's approved and I'd rather not do it when you and Chris are both out.

**Comment by robincornett** [comment]
@ashleyfae re-requesting review due to updating the addon tools please.

File: readme.txt [hunk 2]
```diff
@@ -4,6 +4,13 @@ Allows per-product confirmation pages on successful purchases.
 
 == Changelog ==
 
+= 1.1.8 - December 8, 2021 =
+* Fix: A PHP warning was logged if there were no items in the cart.
+* Dev: Refactor how plugin is loaded
```
Comment (ashleyfae) [inline readme.txt:9]:
```suggestion
* Dev: Refactor how plugin is loaded.
```

I'm surprised at you! :stuck_out_tongue:

**Review by ashleyfae — APPROVED** [review]

**Comment by ashleyfae** [comment]
@robincornett Feel free to release when you're ready.

**Pull request merged**